docs/micros/visor.md

```diff
@@ -1,8 +1,52 @@
-# Visor
-You can view recent logs on visor.
+---
+id: visor
+title: Using Visor
+sidebar_label: Visor
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Visor
+
+Something not working? Use Visor to see a live view of the result of every request processed! Using Visor, you can also test your endpoints by sending your Micro requests!
 
 ### Opening Visor
-Head over to your Deta console to open your micro project; then open the *visor* tab.
+<Tabs 
+    defaultValue="browser" 
+    values={[
+    { label: 'Browser', value: 'browser' },
+    { label: 'Terminal', value: 'terminal' },
+    ]
+}>
+<TabItem value="browser">
+
+Start off by logging into your console at [deta.sh](https://deta.sh).
+
+Once that's done, choose a micro to view by clicking on itname (under the _"Micros"_ tab).

```

Comment:
`Once that's done, choose a micro to view by clicking on its name (under the _"Micros"_ tab).`

Reply:
Good spot, thanks!